tests/playwright-test/ui-mode-fixtures.ts

```diff
@@ -21,17 +21,24 @@ import type { TestChildProcess } from '../config/commonFixtures';
 import { rimraf } from '../../packages/playwright-core/lib/utilsBundle';
 import { cleanEnv, cliEntrypoint, test as base, writeFiles } from './playwright-test-fixtures';
 import type { Files, RunOptions } from './playwright-test-fixtures';
-import type { Browser, Page, TestInfo } from './stable-test-runner';
+import type { Browser, BrowserType, Page, TestInfo } from './stable-test-runner';
 import { createGuid } from '../../packages/playwright-core/src/utils/crypto';
 
+// We want to have ToT playwright-core here, since we install it's browsers and otherwise don't have the right browser revision.
+const chromium: BrowserType = require('../../packages/playwright-core').chromium;
```

Comment:
I'd inline this comment and require to the line 111. I also wonder how do other `ttest` tests that use `page` work? Is there something special in this case?

Reply:
its the only place where we launch a browser inside playwright-test, all the other launches are inside runInlineTest which use ToT test-runner.